x-pack/legacy/plugins/uptime/server/lib/adapters/monitor_states/elasticsearch_monitor_states_adapter.ts

```diff
@@ -92,3 +109,45 @@ const jsonifyPagination = (p: any): string | null => {
 
   return JSON.stringify(p);
 };
+
+const fastStatusCount = async (context: QueryContext): Promise<Snapshot> => {
+  const params = {
+    index: INDEX_NAMES.HEARTBEAT,
+    body: {
+      size: 0,
+      query: { bool: { filter: await context.dateAndCustomFilters() } },
+      aggs: {
+        unique: {
+          cardinality: { field: 'monitor.id', precision_threshold: 40000 },
```

Comment:
I'll add a comment here. 40k is actually the max precision the `cardinality` aggregation supports. We want max precision here, and the storage tradeoff is fine since we don't have a ton of buckets.

Reply:
Commented added in https://github.com/elastic/kibana/pull/52433/commits/5d789d024ab8ace0b1bd406a3c47f9c59e5a2382#diff-d51e4dd872235464063c1c97819da10dR121

Reply:
> 40k is actually the max precision the cardinality aggregation supports

Yep, that's why I wanted some indication in case the reader of the code is unaware; this way they'll understand the choice and not think it's some arbitrary magic number. 👍